packages/charting/src/components/VerticalBarChart/__snapshots__/VerticalBarChart.test.tsx.snap

```diff
@@ -13,55 +13,322 @@ exports[`VerticalBarChart renders VerticalBarChart correctly 1`] = `
         width: 690px;
       }
 >
-  <svg
+  <div
     className=
-
-        {
-          box-sizing: content-box;
-          height: 400px;
-          padding-bottom: 20px;
-          padding-left: 20px;
-          padding-right: 20px;
-          padding-top: 20px;
-          width: 650px;
+        ms-FocusZone
+        &:focus {
+          outline: none;
         }
+    data-focuszone-id="FocusZone0"
+    onFocus={[Function]}
+    onKeyDown={[Function]}
+    onMouseDownCapture={[Function]}
   >
-    <g
+    <svg
       className=
 
           {
-            transform: translate(30px, 350px);
+            box-sizing: content-box;
+            height: 400px;
+            padding-bottom: 20px;
+            padding-left: 20px;
+            padding-right: 20px;
+            padding-top: 20px;
+            width: 650px;
           }
-    />
-    <g
-      className=
+    >
+      <g
+        className=
 
-          {
-            transform: translate(23px, 0px);
-          }
-    />
-    <g
+            {
+              transform: translate(30px, 350px);
+            }
+      />
+      <g
+        className=
+
+            {
+              transform: translate(23px, 0px);
+            }
+      />
+      <g
+        className=
+
+            {
+              transform: translate(30px, 0px);
+            }
+      >
+        <rect
+          className=
+
+              {
+                opacity: ;
+              }
+          data-is-focusable={true}
+          fill="rgb(0, 188, 242)"
+          height={0}
+          onBlur={[Function]}
+          onFocus={[Function]}
+          onMouseLeave={[Function]}
+          onMouseOver={[Function]}
+          width={15}
+          x={0}
+          y={350}
+        />
+        <rect
+          className=
+
+              {
+                opacity: ;
+              }
+          data-is-focusable={true}
+          fill="rgb(0, 32, 80)"
+          height={340}
+          onBlur={[Function]}
+          onFocus={[Function]}
+          onMouseLeave={[Function]}
+          onMouseOver={[Function]}
+          width={15}
+          x={585}
+          y={10}
+        />
+      </g>
+    </svg>
+  </div>
+  <div
+    className=
+
+        {
+          margin-left: 35px;
+          margin-top: 8px;
+        }
+  >
+    <div
       className=
 
           {
-            transform: translate(30px, 0px);
+            align-items: center;
+            white-space: nowrap;
+            width: 100%;
           }
```

Comment:
nit: Please format

Reply:
Formated

---
In reply to: [406300260](https://github.com/microsoft/fluentui/pull/12623#discussion_r406300260) [](ancestors = 406300260)

Reply:
This file is auto generated if any change done manual build is getting failed